.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -27,15 +22,14 @@ body:
     attributes:
       label: Expected behavior
       description: A clear and concise description of what you expected to happen.
-    validations:
-      required: true
 
   - type: dropdown
     id: os
     attributes:
       label: Operating system
       description: What OS are you using?
       options:
+        - Any
```

Comment:
If we are going to add `Any` in here we might want to change the description. I think maybe we should just remove Any and make it optional. Wdyt?

Reply:
turned it into optional